frontend/src/common/types/socket/socket.ts

```diff
@@ -0,0 +1,6 @@
+export { type SocketClient } from './socket-client.type';
+export {
+  type ClientToServerEvents,
+  type ServerToClientEvents,
```

Comment:
Not sure that this typing will work for us, looks like it's not as flexible as we need